dotcom-rendering/package.json

```diff
@@ -100,7 +100,7 @@
 		"@types/k6": "0.47.1",
 		"@types/lodash.debounce": "4.0.7",
 		"@types/lodash.get": "4.4.7",
-		"@types/node": "18.18.14",
+		"@types/node": "20.12.2",
```

Comment:
The [patch does not need to match the Node version](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library) and [should be 7](https://www.npmjs.com/package/@types/node/v/20.12.7).